### DISCUSSION TITLE
Dev in the flash

### DISCUSSION THREAD

**Discussion by rayriffy** [creatorsgarten/discussions#4] [Event ideas]
Flash talk ธีมประมาณ stand up comedy มีเวลาพรีเซนต์แค่ 5 นาทีก่อนตัดไมค์ เหมือน JSConf.Asia Lightning Talks

A fast talk event for where any event members can go up to the stand, and give a quick presentation talk for 5 minutes before microphone get cut off. About the same event in JSConf 2019.

Event format (draft)
---

- เริ่มจากการเปิด Call for presentations ให้คนส่งหัวข้อที่อยากพูดมา (อาจจะเป็นพูดสด หรือ Pre-recorded มาเปิดในงานก็ได้)
- คัดเลือกหัวข้อที่น่าสนใจ ปล่อยตารางงาน
- Talks and plenty of breaks

ตัวงาน อาจจะจัดเป็นสไตล์ Tech meetup ได้ เนื่องจาก Talk สั้นอยู่แล้ว งานน่าจะไม่ยาวมาก (อาจจะได้ฟีลประมาณ [PechaKucha](https://en.wikipedia.org/wiki/PechaKucha) หรือ [Ignite Talks](https://en.wikipedia.org/wiki/Ignite_(event))) แต่ถ้ามีหัวข้อเยอะจริงๆ ก็มีโอกาสที่จะจัดเป็น Conference ในอนาคต (สไตล์คล้ายๆ [!!Con](https://bangbangcon.com/))

Concerns
---

- Running timetable for each rounds could be challenging
- Haven't decided yet that member can present any topic, or given by us trough topic randomization.

**Comment by dtinth** [comment]
อยากให้มีงานแบบนี้เหมือนกัน นึกถึง !!Con ที่มี talk 10 นาทีรัวๆ

- [!!Con 2017: HDR Photography in Microsoft Excel?! by Kevin Chen](https://www.youtube.com/watch?v=bkQJdaGGVM8)
- [!!Con 2017: The emoji that Killed Chrome!! by Julian Squires](https://www.youtube.com/watch?v=UE-fJjMasec)
